oneview_redfish_toolkit/blueprints/resource_block.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import logging
+
+from flask import abort
+from flask import Blueprint
+from flask import g
+from flask import Response
+from flask_api import status
+
+from hpOneView.exceptions import HPOneViewException
+from oneview_redfish_toolkit.api.errors import OneViewRedfishError
+from oneview_redfish_toolkit.api.resource_block_computer_system \
+    import ResourceBlockComputerSystem
+from oneview_redfish_toolkit.api.server_hardware_resource_block \
+    import ServerHardwareResourceBlock
+
+
+RESOURCE_BLOCK_URI = "/redfish/v1/CompositionService/ResourceBlocks"
```

Comment:
This constant is very good, but is referenced just by this class and this URI `"/redfish/v1/CompositionService/ResourceBlocks"` is being used by api class, too. Maybe it should be inside the api class and referenced here (or keep inside here, but referenced in api class, I prefer the first option =).

Reply:
done